packages/ui-a11y-content/src/ScreenReaderContent/index.tsx

```diff
@@ -73,8 +79,7 @@ class ScreenReaderContent extends Component<Props> {
     this.props.makeStyles()
   }
 
-  // @ts-expect-error ts-migrate(6133) FIXME: 'prevProps' is declared but its value is never rea... Remove this comment to see the full error message
-  componentDidUpdate(prevProps, prevState, snapshot) {
+  componentDidUpdate() {
     // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
```

Comment:
I didn't touch these warnings for now, these should be handled when typing the emotion package.